Synthesis failed for nodejs-logging-winston

Hello! Autosynth couldn't regenerate nodejs-logging-winston. :broken_heart:

Here's the output from running `synth.py`:

```
event <InotifyEvent: src_path=b'./node_modules/.staging/@types', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@types>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=467, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@typescript-eslint>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@microsoft', wd=52, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@microsoft', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@microsoft>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@mrmlnc', wd=2555, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@mrmlnc', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@mrmlnc>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=43, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=42, mask=IN_DELETE|IN_ISDIR, cookie=0, name=.staging>
[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-05T12_54_46_274Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-05T12_54_46_325Z-debug.log
2020-08-05 05:54:46,361 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-05 05:54:46,381 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-05 05:54:46,431 autosynth [ERROR] > Synthesis failed
2020-08-05 05:54:46,431 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-05 05:54:46,441 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-05 05:54:46,446 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7b12b27f-67f6-4620-8ebe-674a90d07262/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
rc_path=b'./.kokoro/release/publish.cfg', wd=32, mask=IN_ATTRIB, cookie=0, name=publish.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/release/docs.cfg', wd=32, mask=IN_MODIFY, cookie=0, name=docs.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/release/docs.cfg', wd=32, mask=IN_ATTRIB, cookie=0, name=docs.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/samples-test.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=samples-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/samples-test.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=samples-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/samples-test.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=samples-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/samples-test.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=samples-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/lint.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=lint.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/lint.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=lint.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/lint.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=lint.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/lint.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=lint.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/test.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/test.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/common.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=common.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/common.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=common.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/common.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=common.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/system-test.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=system-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/system-test.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=system-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/system-test.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=system-test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/docs.cfg', wd=36, mask=IN_MODIFY, cookie=0, name=docs.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node10/docs.cfg', wd=36, mask=IN_ATTRIB, cookie=0, name=docs.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node12/test.cfg', wd=38, mask=IN_MODIFY, cookie=0, name=test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node12/test.cfg', wd=38, mask=IN_ATTRIB, cookie=0, name=test.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node12/common.cfg', wd=38, mask=IN_MODIFY, cookie=0, name=common.cfg>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node12/common.cfg', wd=38, mask=IN_MODIFY, cookie=0, name=common.cfg>
2020-08-06 04:17:27,272 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./.kokoro/continuous/node12/common.cfg', wd=38, mask=IN_ATTRIB, cookie=0, name=common.cfg>
npm WARN deprecated ts-simple-ast@12.4.0: NOTICE: ts-simple-ast has been renamed to ts-morph and version reset to 1.0.0. Switch at your leisure...
npm WARN deprecated viz.js@1.8.2: no longer supported
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated opn@6.0.0: The package has been renamed to `open`
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@opencensus%2fpropagation-stackdriver - Not found
npm ERR! 404 
npm ERR! 404  '@opencensus/propagation-stackdriver@0.0.22' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of '@google-cloud/logging'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-06T11_17_42_496Z-debug.log
2020-08-06 04:17:42,515 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-06 04:17:42,522 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-06 04:17:42,578 autosynth [ERROR] > Synthesis failed
2020-08-06 04:17:42,579 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-06 04:17:42,591 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 04:17:42,598 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/76bb7f6f-4d47-4888-b0d2-a8761a276cc8/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
BUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/wide-align/node_modules/strip-ansi/package.json', wd=569, mask=IN_MOVED_TO, cookie=161392, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/validator/package.json.2625592291', wd=639, mask=IN_MOVED_FROM, cookie=161393, name=package.json.2625592291>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/validator/package.json', wd=639, mask=IN_MOVED_TO, cookie=161393, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/wide-align/node_modules/string-width/package.json.1657830435', wd=568, mask=IN_MOVED_FROM, cookie=161394, name=package.json.1657830435>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/wide-align/node_modules/string-width/package.json', wd=568, mask=IN_MOVED_TO, cookie=161394, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg-up/package.json.3763134118', wd=641, mask=IN_ATTRIB, cookie=0, name=package.json.3763134118>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/package.json.2204983266', wd=649, mask=IN_ATTRIB, cookie=0, name=package.json.2204983266>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg/package.json.1922424455', wd=642, mask=IN_ATTRIB, cookie=0, name=package.json.1922424455>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/normalize-package-data/package.json.1922998917', wd=643, mask=IN_ATTRIB, cookie=0, name=package.json.1922998917>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-compile-cache/package.json.4116718283', wd=655, mask=IN_ATTRIB, cookie=0, name=package.json.4116718283>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/node_modules/source-map/package.json.3626887360', wd=651, mask=IN_MODIFY, cookie=0, name=package.json.3626887360>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/validate-npm-package-license/package.json.1763457450', wd=645, mask=IN_ATTRIB, cookie=0, name=package.json.1763457450>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg-up/package.json.3763134118', wd=641, mask=IN_MOVED_FROM, cookie=161395, name=package.json.3763134118>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg-up/package.json', wd=641, mask=IN_MOVED_TO, cookie=161395, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/package.json.2204983266', wd=649, mask=IN_ATTRIB, cookie=0, name=package.json.2204983266>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg/package.json.1922424455', wd=642, mask=IN_MOVED_FROM, cookie=161396, name=package.json.1922424455>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/read-pkg/package.json', wd=642, mask=IN_MOVED_TO, cookie=161396, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-compile-cache/package.json.4116718283', wd=655, mask=IN_ATTRIB, cookie=0, name=package.json.4116718283>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/normalize-package-data/package.json.1922998917', wd=643, mask=IN_MOVED_FROM, cookie=161397, name=package.json.1922998917>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/normalize-package-data/package.json', wd=643, mask=IN_MOVED_TO, cookie=161397, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/@opencensus/propagation-stackdriver/package.json.2778651542', wd=657, mask=IN_ATTRIB, cookie=0, name=package.json.2778651542>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/validate-npm-package-license/package.json.1763457450', wd=645, mask=IN_MOVED_FROM, cookie=161398, name=package.json.1763457450>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/validate-npm-package-license/package.json', wd=645, mask=IN_MOVED_TO, cookie=161398, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/package.json.2204983266', wd=649, mask=IN_MOVED_FROM, cookie=161399, name=package.json.2204983266>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/package.json', wd=649, mask=IN_MOVED_TO, cookie=161399, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-compile-cache/package.json.4116718283', wd=655, mask=IN_MOVED_FROM, cookie=161400, name=package.json.4116718283>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-compile-cache/package.json', wd=655, mask=IN_MOVED_TO, cookie=161400, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/node_modules/source-map/package.json.3626887360', wd=651, mask=IN_ATTRIB, cookie=0, name=package.json.3626887360>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/@opencensus/propagation-stackdriver/package.json.2778651542', wd=657, mask=IN_ATTRIB, cookie=0, name=package.json.2778651542>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/v8-to-istanbul/node_modules/source-map/package.json.3626887360', wd=651, mask=IN_ATTRIB, cookie=0, name=package.json.3626887360>
2020-08-06 12:35:32,278 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-06 12:35:32,329 autosynth [ERROR] > Synthesis failed
2020-08-06 12:35:32,329 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-06 12:35:32,343 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 12:35:32,349 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/374d348d-8a6c-4e4c-b1a2-7114e0aa1049/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
de_modules/.staging/@tootallnate', wd=56, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@tootallnate', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@tootallnate>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@types', wd=54, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@types', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@types>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=463, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@typescript-eslint>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@compodoc', wd=212, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@compodoc', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@compodoc>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=43, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=42, mask=IN_DELETE|IN_ISDIR, cookie=0, name=.staging>
[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-07T12_33_32_256Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-07T12_33_32_324Z-debug.log
2020-08-07 05:33:32,365 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-07 05:33:32,455 autosynth [ERROR] > Synthesis failed
2020-08-07 05:33:32,455 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-07 05:33:32,465 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-07 05:33:32,471 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/46f39e56-2d48-4dfc-8fcc-e5cf5f484e53/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ode_modules/.staging/@types', wd=55, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@types', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@types>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=462, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@typescript-eslint', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@typescript-eslint>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@istanbuljs', wd=51, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@istanbuljs', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@istanbuljs>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@microsoft', wd=52, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging/@microsoft', wd=43, mask=IN_DELETE|IN_ISDIR, cookie=0, name=@microsoft>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=43, mask=IN_DELETE_SELF, cookie=0, name=>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/.staging', wd=42, mask=IN_DELETE|IN_ISDIR, cookie=0, name=.staging>
[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-08T19_43_44_839Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-08T19_43_44_896Z-debug.log
2020-08-08 12:43:44,929 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-08 12:43:45,003 autosynth [ERROR] > Synthesis failed
2020-08-08 12:43:45,003 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-08 12:43:45,010 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-08 12:43:45,015 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4de22315-84b1-493d-8da2-dfa7688128f5/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
rc_path=b'./node_modules/snapdragon/README.md', wd=767, mask=IN_CREATE, cookie=0, name=README.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/index.js', wd=767, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/lib/compiler.js', wd=768, mask=IN_CREATE, cookie=0, name=compiler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/lib/source-maps.js', wd=768, mask=IN_CREATE, cookie=0, name=source-maps.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/lib/utils.js', wd=768, mask=IN_CREATE, cookie=0, name=utils.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/lib/parser.js', wd=768, mask=IN_CREATE, cookie=0, name=parser.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/lib/position.js', wd=768, mask=IN_CREATE, cookie=0, name=position.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug', wd=1435, mask=IN_CREATE|IN_ISDIR, cookie=0, name=debug>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/extend-shallow', wd=1445, mask=IN_CREATE|IN_ISDIR, cookie=0, name=extend-shallow>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/ms', wd=1442, mask=IN_CREATE|IN_ISDIR, cookie=0, name=ms>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map', wd=1432, mask=IN_CREATE|IN_ISDIR, cookie=0, name=source-map>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/define-property', wd=1446, mask=IN_CREATE|IN_ISDIR, cookie=0, name=define-property>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src', wd=1436, mask=IN_CREATE|IN_ISDIR, cookie=0, name=src>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/Makefile', wd=1435, mask=IN_CREATE, cookie=0, name=Makefile>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/node.js', wd=1435, mask=IN_CREATE, cookie=0, name=node.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/package.json', wd=1435, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/CHANGELOG.md', wd=1435, mask=IN_CREATE, cookie=0, name=CHANGELOG.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/LICENSE', wd=1435, mask=IN_CREATE, cookie=0, name=LICENSE>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/.npmignore', wd=1435, mask=IN_CREATE, cookie=0, name=.npmignore>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/.eslintrc', wd=1435, mask=IN_CREATE, cookie=0, name=.eslintrc>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/component.json', wd=1435, mask=IN_CREATE, cookie=0, name=component.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/README.md', wd=1435, mask=IN_CREATE, cookie=0, name=README.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/.travis.yml', wd=1435, mask=IN_CREATE, cookie=0, name=.travis.yml>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/.coveralls.yml', wd=1435, mask=IN_CREATE, cookie=0, name=.coveralls.yml>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/karma.conf.js', wd=1435, mask=IN_CREATE, cookie=0, name=karma.conf.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src/node.js', wd=1436, mask=IN_CREATE, cookie=0, name=node.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src/inspector-log.js', wd=1436, mask=IN_CREATE, cookie=0, name=inspector-log.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src/debug.js', wd=1436, mask=IN_CREATE, cookie=0, name=debug.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src/index.js', wd=1436, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/debug/src/browser.js', wd=1436, mask=IN_CREATE, cookie=0, name=browser.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/extend-shallow/package.json', wd=1445, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/extend-shallow/LICENSE', wd=1445, mask=IN_CREATE, cookie=0, name=LICENSE>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/extend-shallow/README.md', wd=1445, mask=IN_CREATE, cookie=0, name=README.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/extend-shallow/index.js', wd=1445, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/ms/package.json', wd=1442, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/ms/readme.md', wd=1442, mask=IN_CREATE, cookie=0, name=readme.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/ms/index.js', wd=1442, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/ms/license.md', wd=1442, mask=IN_CREATE, cookie=0, name=license.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/lib', wd=1433, mask=IN_CREATE|IN_ISDIR, cookie=0, name=lib>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/dist', wd=1434, mask=IN_CREATE|IN_ISDIR, cookie=0, name=dist>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/package.json', wd=1432, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/CHANGELOG.md', wd=1432, mask=IN_CREATE, cookie=0, name=CHANGELOG.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/LICENSE', wd=1432, mask=IN_CREATE, cookie=0, name=LICENSE>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/snapdragon/node_modules/source-map/README.md', wd=1432, mask=IN_CREATE, cookie=0, name=README.md>
2020-08-09 06:26:01,241 autosynth [ERROR] > Synthesis failed
2020-08-09 06:26:01,241 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-09 06:26:01,249 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-09 06:26:01,253 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/96fb0e9d-e02a-4451-878f-e646368369cc/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
201, mask=IN_CREATE, cookie=0, name=create-logger.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/tail-file.js', wd=201, mask=IN_CREATE, cookie=0, name=tail-file.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/common.js', wd=201, mask=IN_CREATE, cookie=0, name=common.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/profiler.js', wd=201, mask=IN_CREATE, cookie=0, name=profiler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/exception-stream.js', wd=201, mask=IN_CREATE, cookie=0, name=exception-stream.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/rejection-handler.js', wd=201, mask=IN_CREATE, cookie=0, name=rejection-handler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/config/index.d.ts', wd=392, mask=IN_CREATE, cookie=0, name=index.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/config/index.js', wd=392, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/stream.js', wd=257, mask=IN_CREATE, cookie=0, name=stream.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/file.js', wd=257, mask=IN_CREATE, cookie=0, name=file.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/index.d.ts', wd=257, mask=IN_CREATE, cookie=0, name=index.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/index.js', wd=257, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/http.js', wd=257, mask=IN_CREATE, cookie=0, name=http.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/lib/winston/transports/console.js', wd=257, mask=IN_CREATE, cookie=0, name=console.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston', wd=155, mask=IN_CREATE|IN_ISDIR, cookie=0, name=winston>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston.js', wd=154, mask=IN_CREATE, cookie=0, name=winston.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/config', wd=374, mask=IN_CREATE|IN_ISDIR, cookie=0, name=config>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports', wd=242, mask=IN_CREATE|IN_ISDIR, cookie=0, name=transports>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/logger.js', wd=155, mask=IN_CREATE, cookie=0, name=logger.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/container.js', wd=155, mask=IN_CREATE, cookie=0, name=container.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/exception-handler.js', wd=155, mask=IN_CREATE, cookie=0, name=exception-handler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/create-logger.js', wd=155, mask=IN_CREATE, cookie=0, name=create-logger.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/tail-file.js', wd=155, mask=IN_CREATE, cookie=0, name=tail-file.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/common.js', wd=155, mask=IN_CREATE, cookie=0, name=common.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/profiler.js', wd=155, mask=IN_CREATE, cookie=0, name=profiler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/exception-stream.js', wd=155, mask=IN_CREATE, cookie=0, name=exception-stream.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/rejection-handler.js', wd=155, mask=IN_CREATE, cookie=0, name=rejection-handler.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/config/index.js', wd=374, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports/stream.js', wd=242, mask=IN_CREATE, cookie=0, name=stream.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports/file.js', wd=242, mask=IN_CREATE, cookie=0, name=file.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports/index.js', wd=242, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports/http.js', wd=242, mask=IN_CREATE, cookie=0, name=http.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/dist/winston/transports/console.js', wd=242, mask=IN_CREATE, cookie=0, name=console.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream', wd=568, mask=IN_CREATE|IN_ISDIR, cookie=0, name=readable-stream>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/lib', wd=569, mask=IN_CREATE|IN_ISDIR, cookie=0, name=lib>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/package.json', wd=568, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/GOVERNANCE.md', wd=568, mask=IN_CREATE, cookie=0, name=GOVERNANCE.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/LICENSE', wd=568, mask=IN_CREATE, cookie=0, name=LICENSE>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/readable.js', wd=568, mask=IN_CREATE, cookie=0, name=readable.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/experimentalWarning.js', wd=568, mask=IN_CREATE, cookie=0, name=experimentalWarning.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/README.md', wd=568, mask=IN_CREATE, cookie=0, name=README.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/CONTRIBUTING.md', wd=568, mask=IN_CREATE, cookie=0, name=CONTRIBUTING.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/errors-browser.js', wd=568, mask=IN_CREATE, cookie=0, name=errors-browser.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/winston/node_modules/readable-stream/readable-browser.js', wd=568, mask=IN_CREATE, cookie=0, name=readable-browser.js>
2020-08-10 06:31:26,902 autosynth [ERROR] > Synthesis failed
2020-08-10 06:31:26,902 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-10 06:31:26,910 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-10 06:31:26,915 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/44b92241-aefd-4fc5-a5a5-a217263f7395/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/lib/ja/diagnosticMessages.generated.json', wd=1097, mask=IN_CREATE, cookie=0, name=diagnosticMessages.generated.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/lib/it/diagnosticMessages.generated.json', wd=1098, mask=IN_CREATE, cookie=0, name=diagnosticMessages.generated.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/lib/de/diagnosticMessages.generated.json', wd=1099, mask=IN_CREATE, cookie=0, name=diagnosticMessages.generated.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/lib/pt-br/diagnosticMessages.generated.json', wd=2858, mask=IN_CREATE, cookie=0, name=diagnosticMessages.generated.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/lib/ru/diagnosticMessages.generated.json', wd=2889, mask=IN_CREATE, cookie=0, name=diagnosticMessages.generated.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/bin/tsserver', wd=1092, mask=IN_CREATE, cookie=0, name=tsserver>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/typescript/bin/tsc', wd=1092, mask=IN_CREATE, cookie=0, name=tsc>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib', wd=837, mask=IN_CREATE|IN_ISDIR, cookie=0, name=lib>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/docs', wd=838, mask=IN_CREATE|IN_ISDIR, cookie=0, name=docs>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/package.json', wd=836, mask=IN_CREATE, cookie=0, name=package.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/CHANGELOG.md', wd=836, mask=IN_CREATE, cookie=0, name=CHANGELOG.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/LICENSE', wd=836, mask=IN_CREATE, cookie=0, name=LICENSE>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/README.md', wd=836, mask=IN_CREATE, cookie=0, name=README.md>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/empty', wd=839, mask=IN_CREATE|IN_ISDIR, cookie=0, name=empty>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/json', wd=840, mask=IN_CREATE|IN_ISDIR, cookie=0, name=json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/fs', wd=841, mask=IN_CREATE|IN_ISDIR, cookie=0, name=fs>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/path-exists', wd=842, mask=IN_CREATE|IN_ISDIR, cookie=0, name=path-exists>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure', wd=843, mask=IN_CREATE|IN_ISDIR, cookie=0, name=ensure>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/util', wd=844, mask=IN_CREATE|IN_ISDIR, cookie=0, name=util>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/output', wd=845, mask=IN_CREATE|IN_ISDIR, cookie=0, name=output>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/move-sync', wd=846, mask=IN_CREATE|IN_ISDIR, cookie=0, name=move-sync>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/move', wd=847, mask=IN_CREATE|IN_ISDIR, cookie=0, name=move>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/copy', wd=848, mask=IN_CREATE|IN_ISDIR, cookie=0, name=copy>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/copy-sync', wd=849, mask=IN_CREATE|IN_ISDIR, cookie=0, name=copy-sync>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/remove', wd=850, mask=IN_CREATE|IN_ISDIR, cookie=0, name=remove>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/mkdirs', wd=851, mask=IN_CREATE|IN_ISDIR, cookie=0, name=mkdirs>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/index.js', wd=837, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/empty/index.js', wd=839, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/json/index.js', wd=840, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/json/output-json.js', wd=840, mask=IN_CREATE, cookie=0, name=output-json.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/json/output-json-sync.js', wd=840, mask=IN_CREATE, cookie=0, name=output-json-sync.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/json/jsonfile.js', wd=840, mask=IN_CREATE, cookie=0, name=jsonfile.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/fs/index.js', wd=841, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/path-exists/index.js', wd=842, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/file.js', wd=843, mask=IN_CREATE, cookie=0, name=file.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/symlink-paths.js', wd=843, mask=IN_CREATE, cookie=0, name=symlink-paths.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/index.js', wd=843, mask=IN_CREATE, cookie=0, name=index.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/symlink.js', wd=843, mask=IN_CREATE, cookie=0, name=symlink.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/link.js', wd=843, mask=IN_CREATE, cookie=0, name=link.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/ensure/symlink-type.js', wd=843, mask=IN_CREATE, cookie=0, name=symlink-type.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/ts-simple-ast/node_modules/fs-extra/lib/util/utimes.js', wd=844, mask=IN_CREATE, cookie=0, name=utimes.js>
2020-08-12 08:38:00,762 autosynth [ERROR] > Synthesis failed
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
Switched to branch 'autosynth'
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b742586e-df31-4aac-8092-78288e9ea8e7/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-13T16_22_52_266Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-13T16_22_52_328Z-debug.log
2020-08-13 09:22:52,366 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-13 09:22:52,451 autosynth [ERROR] > Synthesis failed
2020-08-13 09:22:52,452 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-13 09:22:52,460 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-13 09:22:52,465 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c36c6dbc-ab79-4f17-b70b-523b420b2a70/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/lydell/resolve-url#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-14T13_38_28_798Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-14T13_38_28_856Z-debug.log
2020-08-14 06:38:28,888 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-14 06:38:28,971 autosynth [ERROR] > Synthesis failed
2020-08-14 06:38:28,971 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-14 06:38:28,979 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-14 06:38:28,984 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4895db2c-aba9-406e-b81d-1e2251ee464c/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-15T13_31_17_655Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-15T13_31_17_729Z-debug.log
2020-08-15 06:31:17,768 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-15 06:31:17,854 autosynth [ERROR] > Synthesis failed
2020-08-15 06:31:17,855 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-15 06:31:17,862 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-15 06:31:17,867 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/756bdf8c-5618-46f2-a657-32fad56d07b8/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-16T12_59_22_164Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-16T12_59_22_233Z-debug.log
2020-08-16 05:59:22,265 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-16 05:59:22,378 autosynth [ERROR] > Synthesis failed
2020-08-16 05:59:22,378 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-16 05:59:22,391 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-16 05:59:22,397 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b32b9cf9-5eaa-4b06-a4b8-6c29d41c8f4e/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-17T12_46_03_673Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-17T12_46_03_754Z-debug.log
2020-08-17 05:46:03,792 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-17 05:46:03,896 autosynth [ERROR] > Synthesis failed
2020-08-17 05:46:03,896 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-17 05:46:03,903 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-17 05:46:03,908 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b89cba32-b27a-436c-8a55-80dd450ca006/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-18T12_49_50_747Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-18T12_49_50_797Z-debug.log
2020-08-18 05:49:50,830 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-18 05:49:50,914 autosynth [ERROR] > Synthesis failed
2020-08-18 05:49:50,915 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-18 05:49:50,923 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-18 05:49:50,929 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/15e3b9fd-f00a-4ff5-9786-abfb8d921425/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-19T13_12_05_176Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-19T13_12_05_231Z-debug.log
2020-08-19 06:12:05,266 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-19 06:12:05,347 autosynth [ERROR] > Synthesis failed
2020-08-19 06:12:05,347 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-19 06:12:05,355 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-19 06:12:05,361 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7a1b0b96-8ddb-4836-a1a2-d2f73b7e6ffe/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
ttps://github.com/lydell/urix#deprecated

> core-js@2.6.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

[96mThank you for using core-js ([94m https://github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-20T13_30_18_654Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-20T13_30_18_721Z-debug.log
2020-08-20 06:30:18,762 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-20 06:30:18,867 autosynth [ERROR] > Synthesis failed
2020-08-20 06:30:18,868 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-20 06:30:18,875 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-20 06:30:18,880 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/68816803-c63c-41b0-87ee-62f4fe249026/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-24T13_24_02_711Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-24T13_24_02_793Z-debug.log
2020-08-24 06:24:02,836 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-24 06:24:02,953 autosynth [ERROR] > Synthesis failed
2020-08-24 06:24:02,954 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-24 06:24:02,961 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-24 06:24:02,967 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/79e2de4e-d268-4d75-9b36-de2d2f477f94/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-25T13_23_57_028Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-25T13_23_57_079Z-debug.log
2020-08-25 06:23:57,116 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-25 06:23:57,206 autosynth [ERROR] > Synthesis failed
2020-08-25 06:23:57,206 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-25 06:23:57,214 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-25 06:23:57,220 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/27324300-305e-44ae-b40c-ba5a81cf88b7/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-26T13_31_26_980Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-26T13_31_27_039Z-debug.log
2020-08-26 06:31:27,072 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-26 06:31:27,175 autosynth [ERROR] > Synthesis failed
2020-08-26 06:31:27,176 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-26 06:31:27,184 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-26 06:31:27,190 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/be27b6c5-61ee-4a18-bb69-0effea9ae23f/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-26T20_57_42_026Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-26T20_57_42_107Z-debug.log
2020-08-26 13:57:42,143 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-26 13:57:42,316 autosynth [ERROR] > Synthesis failed
2020-08-26 13:57:42,316 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-26 13:57:42,324 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-26 13:57:42,329 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4813454c-4e1e-44ef-bc71-454baf0ea50b/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-27T11_43_17_639Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-27T11_43_17_695Z-debug.log
2020-08-27 04:43:17,724 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-27 04:43:17,797 autosynth [ERROR] > Synthesis failed
2020-08-27 04:43:17,798 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-27 04:43:17,805 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-27 04:43:17,810 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a8c1e885-336c-4636-8e00-e4134524f8ee/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-28T11_39_47_616Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-28T11_39_47_675Z-debug.log
2020-08-28 04:39:47,708 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-28 04:39:47,787 autosynth [ERROR] > Synthesis failed
2020-08-28 04:39:47,787 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-28 04:39:47,794 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-28 04:39:47,799 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/cd5e1e45-53d4-4611-af5a-f0b4ec467792/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-29T11_39_51_288Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-29T11_39_51_338Z-debug.log
2020-08-29 04:39:51,371 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-29 04:39:51,449 autosynth [ERROR] > Synthesis failed
2020-08-29 04:39:51,450 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-29 04:39:51,457 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-29 04:39:51,462 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/313180d7-89ba-4335-ad20-0c665ae09648/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-30T11_39_32_317Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-30T11_39_32_370Z-debug.log
2020-08-30 04:39:32,397 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-30 04:39:32,460 autosynth [ERROR] > Synthesis failed
2020-08-30 04:39:32,461 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-30 04:39:32,467 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-30 04:39:32,471 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/752224c8-6616-4d30-9cf4-227a8be67f96/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-31T11_39_27_089Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-31T11_39_27_138Z-debug.log
2020-08-31 04:39:27,169 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-31 04:39:27,246 autosynth [ERROR] > Synthesis failed
2020-08-31 04:39:27,247 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-08-31 04:39:27,253 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-31 04:39:27,258 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/52b76160-53d6-44db-8e1e-19f9e6e24cdf/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-01T11_39_52_012Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-01T11_39_52_078Z-debug.log
2020-09-01 04:39:52,107 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-01 04:39:52,193 autosynth [ERROR] > Synthesis failed
2020-09-01 04:39:52,193 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-01 04:39:52,202 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-01 04:39:52,207 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/479f0207-8337-4bbf-8997-4939c834f4ff/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-02T11_39_22_057Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-02T11_39_22_112Z-debug.log
2020-09-02 04:39:22,142 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-02 04:39:22,216 autosynth [ERROR] > Synthesis failed
2020-09-02 04:39:22,217 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-02 04:39:22,223 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-02 04:39:22,227 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/486e753c-3c0c-435e-a910-08de8fa2ca10/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-03T11_39_51_691Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-03T11_39_51_768Z-debug.log
2020-09-03 04:39:51,901 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-03 04:39:52,024 autosynth [ERROR] > Synthesis failed
2020-09-03 04:39:52,024 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-03 04:39:52,037 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-03 04:39:52,041 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c1f9d459-507f-4eff-b2b0-b2b831610431/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-04T11_39_40_218Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-04T11_39_40_269Z-debug.log
2020-09-04 04:39:40,300 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-04 04:39:40,377 autosynth [ERROR] > Synthesis failed
2020-09-04 04:39:40,378 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-04 04:39:40,385 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-04 04:39:40,390 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c5f53f7e-7825-486a-b6e6-dea34088685d/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-05T11_39_18_746Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-05T11_39_18_803Z-debug.log
2020-09-05 04:39:18,831 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-05 04:39:18,899 autosynth [ERROR] > Synthesis failed
2020-09-05 04:39:18,899 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-05 04:39:18,910 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-05 04:39:18,915 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7e52414f-eded-453e-a82d-c40c0dfe0194/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-06T11_39_42_655Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-06T11_39_42_710Z-debug.log
2020-09-06 04:39:42,739 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-06 04:39:42,805 autosynth [ERROR] > Synthesis failed
2020-09-06 04:39:42,805 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-06 04:39:42,812 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-06 04:39:42,817 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1fe3a080-3469-40fc-a9cc-a80d08598a88/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-07T11_40_04_680Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-07T11_40_04_756Z-debug.log
2020-09-07 04:40:04,789 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-07 04:40:04,907 autosynth [ERROR] > Synthesis failed
2020-09-07 04:40:04,907 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-07 04:40:04,923 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-07 04:40:04,978 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1c30da0d-a460-4d67-9a11-824fb97a664a/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-08T11_39_46_729Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-08T11_39_46_785Z-debug.log
2020-09-08 04:39:46,816 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-08 04:39:46,895 autosynth [ERROR] > Synthesis failed
2020-09-08 04:39:46,895 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-08 04:39:46,903 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-08 04:39:46,908 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7cbf8241-e95e-4417-943d-32148d4b910a/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-09T11_39_28_305Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-09T11_39_28_362Z-debug.log
2020-09-09 04:39:28,394 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-09 04:39:28,477 autosynth [ERROR] > Synthesis failed
2020-09-09 04:39:28,477 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-09 04:39:28,484 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 04:39:28,489 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/aaea0b66-d353-42ea-acb1-d6bc7601f62f/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-10T11_39_33_688Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-10T11_39_33_744Z-debug.log
2020-09-10 04:39:33,774 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-10 04:39:33,853 autosynth [ERROR] > Synthesis failed
2020-09-10 04:39:33,853 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-10 04:39:33,860 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 04:39:33,865 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d4829e8c-11e1-4022-89ce-8fceff9c7aa3/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-11T11_39_27_300Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-11T11_39_27_381Z-debug.log
2020-09-11 04:39:27,427 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-11 04:39:27,514 autosynth [ERROR] > Synthesis failed
2020-09-11 04:39:27,514 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-11 04:39:27,523 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 04:39:27,528 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c9a6f0ce-5e1a-42a6-8f18-c290999c0d0c/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-12T11_43_37_283Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-12T11_43_37_342Z-debug.log
2020-09-12 04:43:37,381 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-12 04:43:37,519 autosynth [ERROR] > Synthesis failed
2020-09-12 04:43:37,519 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-12 04:43:37,526 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 04:43:37,531 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0832b96d-f20d-428a-9244-9373d24004a9/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-13T11_39_26_002Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-13T11_39_26_070Z-debug.log
2020-09-13 04:39:26,106 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-13 04:39:26,241 autosynth [ERROR] > Synthesis failed
2020-09-13 04:39:26,241 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-13 04:39:26,252 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 04:39:26,257 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fe45012c-d1e6-4302-9072-f64ff327d183/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-14T11_39_39_486Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-14T11_39_39_553Z-debug.log
2020-09-14 04:39:39,595 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-14 04:39:39,743 autosynth [ERROR] > Synthesis failed
2020-09-14 04:39:39,744 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-14 04:39:39,751 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 04:39:39,756 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/19f33592-a008-4724-9651-05cd20169de7/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-15T11_39_57_541Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-15T11_39_57_611Z-debug.log
2020-09-15 04:39:57,654 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-15 04:39:57,802 autosynth [ERROR] > Synthesis failed
2020-09-15 04:39:57,803 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-15 04:39:57,810 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 04:39:57,816 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/38f69b41-c799-41f4-b3c9-38b72868b1db/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-16T11_39_21_712Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-16T11_39_21_779Z-debug.log
2020-09-16 04:39:21,811 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-16 04:39:21,963 autosynth [ERROR] > Synthesis failed
2020-09-16 04:39:21,964 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-16 04:39:21,975 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 04:39:21,980 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bf3ad9e3-fb60-4f0a-a997-848e063a074b/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

Autosynth is still having trouble generating nodejs-logging-winston. :sob:

Here's the output from running `synth.py`:

```
github.com/zloirock/core-js [96m) for polyfilling JavaScript standard library![0m

[96mThe project needs your help! Please consider supporting of core-js on Open Collective or Patreon: [0m
[96m>[94m https://opencollective.com/core-js [0m
[96m>[94m https://www.patreon.com/zloirock [0m

[96mAlso, the author of core-js ([94m https://github.com/zloirock [96m) is looking for a good job -)[0m


> protobufjs@6.10.1 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/protobufjs
> node scripts/postinstall


> @compodoc/compodoc@1.1.11 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging-winston/node_modules/@compodoc/compodoc
> opencollective-postinstall || exit 0

[96m[1mThank you for using @compodoc/compodoc![96m[1m
[0m[96mIf you rely on this package, please consider supporting our open collective:[22m[39m
> [94mhttps://opencollective.com/compodoc/donate[0m


> @google-cloud/logging-winston@4.0.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> npm run compile


> @google-cloud/logging-winston@4.0.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> gts clean

version: 12
Removing build ...

> @google-cloud/logging-winston@4.0.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging-winston
> tsc -p .

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-17T11_39_35_850Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging-winston@4.0.0 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging-winston@4.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-09-17T11_39_35_919Z-debug.log
2020-09-17 04:39:35,958 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging-winston/synth.py", line 27, in <module>
    node.install()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-09-17 04:39:36,060 autosynth [ERROR] > Synthesis failed
2020-09-17 04:39:36,061 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 4abe705 chore: add dev dependencies for cloud-rad ref docs (#527)
2020-09-17 04:39:36,068 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 04:39:36,074 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/continuous/node12/lint.cfg
Removing .kokoro/continuous/node12/samples-test.cfg
Removing .kokoro/continuous/node12/system-test.cfg
Removing .kokoro/presubmit/node12/samples-test.cfg
Removing .kokoro/presubmit/node12/system-test.cfg
Removing .kokoro/release/docs-devsite.cfg
Removing .kokoro/release/docs-devsite.sh
Removing __pycache__/
Removing api-extractor.json
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d073ca01-b18e-4d1e-b56c-ccf2397eed56/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging-winston;failed=false).

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,3 +1,4 @@
 *.ts text eol=lf
 *.js text eol=lf
 protos/* linguist-generated
+**/api-extractor.json  linguist-language=JSON-with-Comments

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -16,7 +16,7 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - run: node --version
-      - run: npm install
+      - run: npm install --engine-strict
       - run: npm test
       - name: coverage
         uses: codecov/codecov-action@v1

--- a/.kokoro/continuous/node12/lint.cfg
+++ b/.kokoro/continuous/node12/lint.cfg
@@ -1,0 +1,4 @@
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/lint.sh"
+}

--- a/.kokoro/continuous/node12/samples-test.cfg
+++ b/.kokoro/continuous/node12/samples-test.cfg
@@ -1,0 +1,7 @@
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/samples-test.sh"
+}

--- a/.kokoro/continuous/node12/system-test.cfg
+++ b/.kokoro/continuous/node12/system-test.cfg
@@ -1,0 +1,7 @@
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/system-test.sh"
+}

--- a/.kokoro/populate-secrets.sh
+++ b/.kokoro/populate-secrets.sh
@@ -32,7 +32,6 @@ do
     --volume=${KOKORO_GFILE_DIR}:${KOKORO_GFILE_DIR} \
     gcr.io/google.com/cloudsdktool/cloud-sdk \
     secrets versions access latest \
-    --credential-file-override=${KOKORO_GFILE_DIR}/kokoro-trampoline.service-account.json \
     --project cloud-devrel-kokoro-resources \
     --secret $key > \
     "$SECRET_LOCATION/$key"

--- a/.kokoro/presubmit/node12/samples-test.cfg
+++ b/.kokoro/presubmit/node12/samples-test.cfg
@@ -1,0 +1,7 @@
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/samples-test.sh"
+}

--- a/.kokoro/presubmit/node12/system-test.cfg
+++ b/.kokoro/presubmit/node12/system-test.cfg
@@ -1,0 +1,7 @@
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/system-test.sh"
+}

--- a/.kokoro/release/docs-devsite.cfg
+++ b/.kokoro/release/docs-devsite.cfg
@@ -1,0 +1,26 @@
+# service account used to publish up-to-date docs.
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 73713
+      keyname: "docuploader_service_account"
+    }
+  }
+}
+
+# doc publications use a Python image.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:10-user"
+}
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Use the trampoline script to run in docker.
+build_file: "nodejs-logging-winston/.kokoro/trampoline.sh"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-logging-winston/.kokoro/release/docs-devsite.sh"
+}

--- a/.kokoro/release/docs-devsite.sh
+++ b/.kokoro/release/docs-devsite.sh
@@ -1,0 +1,67 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# build jsdocs (Python is installed on the Node 10 docker image).
+if [[ -z "$CREDENTIALS" ]]; then
+  # if CREDENTIALS are explicitly set, assume we're testing locally
+  # and don't set NPM_CONFIG_PREFIX.
+  export NPM_CONFIG_PREFIX=/home/node/.npm-global
+  export PATH="$PATH:/home/node/.npm-global/bin"
+  cd $(dirname $0)/../..
+fi
+
+mkdir ./etc
+
+npm install
+npm run api-extractor
+npm run api-documenter
+
+npm i json@9.0.6 -g
+NAME=$(cat .repo-metadata.json | json name)
+
+mkdir ./_devsite
+cp ./yaml/$NAME/* ./_devsite
+
+# Delete SharePoint item, see https://github.com/microsoft/rushstack/issues/1229
+sed -i -e '1,3d' ./yaml/toc.yml
+sed -i -e 's/^    //' ./yaml/toc.yml
+
+cp ./yaml/toc.yml ./_devsite/toc.yml
+
+# create docs.metadata, based on package.json and .repo-metadata.json.
+pip install -U pip
+python3 -m pip install --user gcp-docuploader
+python3 -m docuploader create-metadata \
+  --name=$NAME \
+  --version=$(cat package.json | json version) \
+  --language=$(cat .repo-metadata.json | json language) \
+  --distribution-name=$(cat .repo-metadata.json | json distribution_name) \
+  --product-page=$(cat .repo-metadata.json | json product_documentation) \
+  --github-repository=$(cat .repo-metadata.json | json repo) \
+  --issue-tracker=$(cat .repo-metadata.json | json issue_tracker)
+cp docs.metadata ./_devsite/docs.metadata
+
+# deploy the docs.
+if [[ -z "$CREDENTIALS" ]]; then
+  CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73713_docuploader_service_account
+fi
+if [[ -z "$BUCKET" ]]; then
+  BUCKET=docs-staging-v2-staging
+fi
+
+python3 -m docuploader upload ./_devsite --destination-prefix docfx --credentials $CREDENTIALS --staging-bucket $BUCKET

--- a/.kokoro/release/publish.cfg
+++ b/.kokoro/release/publish.cfg
@@ -49,7 +49,7 @@ before_action {
 
 env_vars: {
   key: "SECRET_MANAGER_KEYS"
-  value: "npm_publish_token"
+  value: "npm_publish_token,releasetool-publish-reporter-app,releasetool-publish-reporter-googleapis-installation,releasetool-publish-reporter-pem"
 }
 
 # Download trampoline resources.
@@ -61,7 +61,7 @@ build_file: "nodejs-logging-winston/.kokoro/trampoline.sh"
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
     key: "TRAMPOLINE_IMAGE"
-    value: "gcr.io/cloud-devrel-kokoro-resources/node:8-user"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:12-user"
 }
 
 env_vars: {

--- a/.kokoro/samples-test.sh
+++ b/.kokoro/samples-test.sh
@@ -41,7 +41,7 @@ if [ -f samples/package.json ]; then
     cd ..
     # If tests are running against master, configure Build Cop
     # to open issues on failures:
-    if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
+    if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]] || [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"nightly"* ]]; then
       export MOCHA_REPORTER_OUTPUT=test_output_sponge_log.xml
       export MOCHA_REPORTER=xunit
       cleanup() {

--- a/.kokoro/system-test.sh
+++ b/.kokoro/system-test.sh
@@ -35,7 +35,7 @@ npm install
 
 # If tests are running against master, configure Build Cop
 # to open issues on failures:
-if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]] || [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"nightly"* ]]; then
   export MOCHA_REPORTER_OUTPUT=test_output_sponge_log.xml
   export MOCHA_REPORTER=xunit
   cleanup() {

--- a/.kokoro/test.sh
+++ b/.kokoro/test.sh
@@ -23,7 +23,7 @@ cd $(dirname $0)/..
 npm install
 # If tests are running against master, configure Build Cop
 # to open issues on failures:
-if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]] || [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"nightly"* ]]; then
   export MOCHA_REPORTER_OUTPUT=test_output_sponge_log.xml
   export MOCHA_REPORTER=xunit
   cleanup() {

--- a/.mocharc.js
+++ b/.mocharc.js
@@ -14,7 +14,8 @@
 const config = {
   "enable-source-maps": true,
   "throw-deprecation": true,
-  "timeout": 10000
+  "timeout": 10000,
+  "recursive": true
 }
 if (process.env.MOCHA_THROW_DEPRECATION === 'false') {
   delete config['throw-deprecation'];

--- a/README.md
+++ b/README.md
@@ -17,6 +17,9 @@ This module provides a higher-level layer for working with
 transport to your existing Winston loggers.
 
 
+A comprehensive list of changes in each version may be found in
+[the CHANGELOG](https://github.com/googleapis/nodejs-logging-winston/blob/master/CHANGELOG.md).
+
 * [Stackdriver Logging for Winston Node.js Client API Reference][client-docs]
 * [Stackdriver Logging for Winston Documentation][product-docs]
 * [github.com/googleapis/nodejs-logging-winston](https://github.com/googleapis/nodejs-logging-winston)

--- a/api-extractor.json
+++ b/api-extractor.json
@@ -1,0 +1,369 @@
+/** 
+ * Config file for API Extractor.  For more info, please visit: https://api-extractor.com
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
+  /**
+   * Optionally specifies another JSON config file that this file extends from.  This provides a way for
+   * standard settings to be shared across multiple projects.
+   *
+   * If the path starts with "./" or "../", the path is resolved relative to the folder of the file that contains
+   * the "extends" field.  Otherwise, the first path segment is interpreted as an NPM package name, and will be
+   * resolved using NodeJS require().
+   *
+   * SUPPORTED TOKENS: none
+   * DEFAULT VALUE: ""
+   */
+  // "extends": "./shared/api-extractor-base.json"
+  // "extends": "my-package/include/api-extractor-base.json"
+
+  /**
+   * Determines the "<projectFolder>" token that can be used with other config file settings.  The project folder
+   * typically contains the tsconfig.json and package.json config files, but the path is user-defined.
+   *
+   * The path is resolved relative to the folder of the config file that contains the setting.
+   *
+   * The default value for "projectFolder" is the token "<lookup>", which means the folder is determined by traversing
+   * parent folders, starting from the folder containing api-extractor.json, and stopping at the first folder
+   * that contains a tsconfig.json file.  If a tsconfig.json file cannot be found in this way, then an error
+   * will be reported.
+   *
+   * SUPPORTED TOKENS: <lookup>
+   * DEFAULT VALUE: "<lookup>"
+   */
+  // "projectFolder": "..",
+
+  /**
+   * (REQUIRED) Specifies the .d.ts file to be used as the starting point for analysis.  API Extractor
+   * analyzes the symbols exported by this module.
+   *
+   * The file extension must be ".d.ts" and not ".ts".
+   *
+   * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+   * prepend a folder token such as "<projectFolder>".
+   *
+   * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+   */
+  "mainEntryPointFilePath": "<projectFolder>/protos/protos.d.ts",
+
+  /**
+   * A list of NPM package names whose exports should be treated as part of this package.
+   *
+   * For example, suppose that Webpack is used to generate a distributed bundle for the project "library1",
+   * and another NPM package "library2" is embedded in this bundle.  Some types from library2 may become part
+   * of the exported API for library1, but by default API Extractor would generate a .d.ts rollup that explicitly
+   * imports library2.  To avoid this, we can specify:
+   *
+   *   "bundledPackages": [ "library2" ],
+   *
+   * This would direct API Extractor to embed those types directly in the .d.ts rollup, as if they had been
+   * local files for library1.
+   */
+  "bundledPackages": [ ],
+
+  /**
+   * Determines how the TypeScript compiler engine will be invoked by API Extractor.
+   */
+  "compiler": {
+    /**
+     * Specifies the path to the tsconfig.json file to be used by API Extractor when analyzing the project.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * Note: This setting will be ignored if "overrideTsconfig" is used.
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<projectFolder>/tsconfig.json"
+     */
+    // "tsconfigFilePath": "<projectFolder>/tsconfig.json",
+
+    /**
+     * Provides a compiler configuration that will be used instead of reading the tsconfig.json file from disk.
+     * The object must conform to the TypeScript tsconfig schema:
+     *
+     * http://json.schemastore.org/tsconfig
+     *
+     * If omitted, then the tsconfig.json file will be read from the "projectFolder".
+     *
+     * DEFAULT VALUE: no overrideTsconfig section
+     */
+    // "overrideTsconfig": {
+    //   . . .
+    // }
+
+    /**
+     * This option causes the compiler to be invoked with the --skipLibCheck option. This option is not recommended
+     * and may cause API Extractor to produce incomplete or incorrect declarations, but it may be required when
+     * dependencies contain declarations that are incompatible with the TypeScript engine that API Extractor uses
+     * for its analysis.  Where possible, the underlying issue should be fixed rather than relying on skipLibCheck.
+     *
+     * DEFAULT VALUE: false
+     */
+    // "skipLibCheck": true,
+  },
+
+  /**
+   * Configures how the API report file (*.api.md) will be generated.
+   */
+  "apiReport": {
+    /**
+     * (REQUIRED) Whether to generate an API report.
+     */
+    "enabled": true,
+
+    /**
+     * The filename for the API report files.  It will be combined with "reportFolder" or "reportTempFolder" to produce
+     * a full file path.
+     *
+     * The file extension should be ".api.md", and the string should not contain a path separator such as "\" or "/".
+     *
+     * SUPPORTED TOKENS: <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<unscopedPackageName>.api.md"
+     */
+    // "reportFileName": "<unscopedPackageName>.api.md",
+
+    /**
+     * Specifies the folder where the API report file is written.  The file name portion is determined by
+     * the "reportFileName" setting.
+     *
+     * The API report file is normally tracked by Git.  Changes to it can be used to trigger a branch policy,
+     * e.g. for an API review.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<projectFolder>/etc/"
+     */
+    // "reportFolder": "<projectFolder>/etc/",
+
+    /**
+     * Specifies the folder where the temporary report file is written.  The file name portion is determined by
+     * the "reportFileName" setting.
+     *
+     * After the temporary file is written to disk, it is compared with the file in the "reportFolder".
+     * If they are different, a production build will fail.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<projectFolder>/temp/"
+     */
+    // "reportTempFolder": "<projectFolder>/temp/"
+  },
+
+  /**
+   * Configures how the doc model file (*.api.json) will be generated.
+   */
+  "docModel": {
+    /**
+     * (REQUIRED) Whether to generate a doc model file.
+     */
+    "enabled": true,
+
+    /**
+     * The output path for the doc model file.  The file extension should be ".api.json".
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<projectFolder>/temp/<unscopedPackageName>.api.json"
+     */
+    // "apiJsonFilePath": "<projectFolder>/temp/<unscopedPackageName>.api.json"
+  },
+
+  /**
+   * Configures how the .d.ts rollup file will be generated.
+   */
+  "dtsRollup": {
+    /**
+     * (REQUIRED) Whether to generate the .d.ts rollup file.
+     */
+    "enabled": true,
+
+    /**
+     * Specifies the output path for a .d.ts rollup file to be generated without any trimming.
+     * This file will include all declarations that are exported by the main entry point.
+     *
+     * If the path is an empty string, then this file will not be written.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<projectFolder>/dist/<unscopedPackageName>.d.ts"
+     */
+    // "untrimmedFilePath": "<projectFolder>/dist/<unscopedPackageName>.d.ts",
+
+    /**
+     * Specifies the output path for a .d.ts rollup file to be generated with trimming for a "beta" release.
+     * This file will include only declarations that are marked as "@public" or "@beta".
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: ""
+     */
+    // "betaTrimmedFilePath": "<projectFolder>/dist/<unscopedPackageName>-beta.d.ts",
+
+
+    /**
+     * Specifies the output path for a .d.ts rollup file to be generated with trimming for a "public" release.
+     * This file will include only declarations that are marked as "@public".
+     *
+     * If the path is an empty string, then this file will not be written.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: ""
+     */
+    // "publicTrimmedFilePath": "<projectFolder>/dist/<unscopedPackageName>-public.d.ts",
+
+    /**
+     * When a declaration is trimmed, by default it will be replaced by a code comment such as
+     * "Excluded from this release type: exampleMember".  Set "omitTrimmingComments" to true to remove the
+     * declaration completely.
+     *
+     * DEFAULT VALUE: false
+     */
+    // "omitTrimmingComments": true
+  },
+
+  /**
+   * Configures how the tsdoc-metadata.json file will be generated.
+   */
+  "tsdocMetadata": {
+    /**
+     * Whether to generate the tsdoc-metadata.json file.
+     *
+     * DEFAULT VALUE: true
+     */
+    // "enabled": true,
+
+    /**
+     * Specifies where the TSDoc metadata file should be written.
+     *
+     * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+     * prepend a folder token such as "<projectFolder>".
+     *
+     * The default value is "<lookup>", which causes the path to be automatically inferred from the "tsdocMetadata",
+     * "typings" or "main" fields of the project's package.json.  If none of these fields are set, the lookup
+     * falls back to "tsdoc-metadata.json" in the package folder.
+     *
+     * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+     * DEFAULT VALUE: "<lookup>"
+     */
+    // "tsdocMetadataFilePath": "<projectFolder>/dist/tsdoc-metadata.json"
+  },
+
+  /**
+   * Specifies what type of newlines API Extractor should use when writing output files.  By default, the output files
+   * will be written with Windows-style newlines.  To use POSIX-style newlines, specify "lf" instead.
+   * To use the OS's default newline kind, specify "os".
+   *
+   * DEFAULT VALUE: "crlf"
+   */
+  // "newlineKind": "crlf",
+
+  /**
+   * Configures how API Extractor reports error and warning messages produced during analysis.
+   *
+   * There are three sources of messages:  compiler messages, API Extractor messages, and TSDoc messages.
+   */
+  "messages": {
+    /**
+     * Configures handling of diagnostic messages reported by the TypeScript compiler engine while analyzing
+     * the input .d.ts files.
+     *
+     * TypeScript message identifiers start with "TS" followed by an integer.  For example: "TS2551"
+     *
+     * DEFAULT VALUE:  A single "default" entry with logLevel=warning.
+     */
+    "compilerMessageReporting": {
+      /**
+       * Configures the default routing for messages that don't match an explicit rule in this table.
+       */
+      "default": {
+        /**
+         * Specifies whether the message should be written to the the tool's output log.  Note that
+         * the "addToApiReportFile" property may supersede this option.
+         *
+         * Possible values: "error", "warning", "none"
+         *
+         * Errors cause the build to fail and return a nonzero exit code.  Warnings cause a production build fail
+         * and return a nonzero exit code.  For a non-production build (e.g. when "api-extractor run" includes
+         * the "--local" option), the warning is displayed but the build will not fail.
+         *
+         * DEFAULT VALUE: "warning"
+         */
+        "logLevel": "warning",
+
+        /**
+         * When addToApiReportFile is true:  If API Extractor is configured to write an API report file (.api.md),
+         * then the message will be written inside that file; otherwise, the message is instead logged according to
+         * the "logLevel" option.
+         *
+         * DEFAULT VALUE: false
+         */
+        // "addToApiReportFile": false
+      },
+
+      // "TS2551": {
+      //   "logLevel": "warning",
+      //   "addToApiReportFile": true
+      // },
+      //
+      // . . .
+    },
+
+    /**
+     * Configures handling of messages reported by API Extractor during its analysis.
+     *
+     * API Extractor message identifiers start with "ae-".  For example: "ae-extra-release-tag"
+     *
+     * DEFAULT VALUE: See api-extractor-defaults.json for the complete table of extractorMessageReporting mappings
+     */
+    "extractorMessageReporting": {
+      "default": {
+        "logLevel": "warning",
+        // "addToApiReportFile": false
+      },
+
+      // "ae-extra-release-tag": {
+      //   "logLevel": "warning",
+      //   "addToApiReportFile": true
+      // },
+      //
+      // . . .
+    },
+
+    /**
+     * Configures handling of messages reported by the TSDoc parser when analyzing code comments.
+     *
+     * TSDoc message identifiers start with "tsdoc-".  For example: "tsdoc-link-tag-unescaped-text"
+     *
+     * DEFAULT VALUE:  A single "default" entry with logLevel=warning.
+     */
+    "tsdocMessageReporting": {
+      "default": {
+        "logLevel": "warning",
+        // "addToApiReportFile": false
+      }
+
+      // "tsdoc-link-tag-unescaped-text": {
+      //   "logLevel": "warning",
+      //   "addToApiReportFile": true
+      // },
+      //
+      // . . .
+    }
+  }
+
+}

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
-    "@google-cloud/logging": "^8.0.0",
+    "@google-cloud/logging": "^8.0.5",
     "google-auth-library": "^6.0.0",
     "lodash.mapvalues": "^4.6.0",
     "logform": "^2.1.2",
@@ -62,6 +62,8 @@
   "devDependencies": {
     "@compodoc/compodoc": "^1.1.10",
     "@google-cloud/common": "^3.0.0",
+    "@microsoft/api-documenter": "^7.8.10",
+    "@microsoft/api-extractor": "^7.8.10",
     "@types/lodash.mapvalues": "^4.6.6",
     "@types/mocha": "^8.0.0",
     "@types/node": "^12.0.7",
@@ -78,11 +80,9 @@
     "mocha": "^8.0.0",
     "post-install-check": "0.0.1",
     "proxyquire": "^2.1.0",
-    "typescript": "3.6.4",
+    "typescript": "^3.9.7",
     "uuid": "^8.0.0",
-    "winston": "^3.2.1",
-    "@microsoft/api-documenter": "^7.8.10",
-    "@microsoft/api-extractor": "^7.8.10"
+    "winston": "^3.2.1"
   },
   "peerDependencies": {
     "winston": ">=3.2.1"

--- a/src/common.ts
+++ b/src/common.ts
@@ -30,7 +30,7 @@ export type MonitoredResource = protos.google.api.MonitoredResource;
 export interface StackdriverData {
   serviceContext?: ServiceContext;
   message?: string;
-  metadata?: Metadata;
+  metadata?: Metadata|MetadataArg;
 }
 
 export interface Metadata {

--- a/src/common.ts
+++ b/src/common.ts
@@ -30,7 +30,7 @@ export type MonitoredResource = protos.google.api.MonitoredResource;
 export interface StackdriverData {
   serviceContext?: ServiceContext;
   message?: string;
-  metadata?: Metadata|MetadataArg;
+  metadata?: Metadata | MetadataArg;
 }
 
 export interface Metadata {

--- a/synth.metadata
+++ b/synth.metadata
@@ -3,16 +3,60 @@
     {
       "git": {
         "name": ".",
-        "remote": "https://github.com/googleapis/nodejs-logging-winston.git",
-        "sha": "0998ad0d51887e637a8791a27d2a4fdb8ff1d394"
-      }
-    },
-    {
-      "git": {
-        "name": "synthtool",
-        "remote": "https://github.com/googleapis/synthtool.git",
-        "sha": "d82deccf657a66e31bd5da9efdb96c6fa322fc7e"
+        "remote": "git@github.com:googleapis/nodejs-logging-winston.git",
+        "sha": "d203f7f32f62a34b3f571025199445142153abe9"
       }
     }
+  ],
+  "generatedFiles": [
+    ".eslintignore",
+    ".eslintrc.json",
+    ".gitattributes",
+    ".github/ISSUE_TEMPLATE/bug_report.md",
+    ".github/ISSUE_TEMPLATE/feature_request.md",
+    ".github/ISSUE_TEMPLATE/support_request.md",
+    ".github/PULL_REQUEST_TEMPLATE.md",
+    ".github/release-please.yml",
+    ".github/workflows/ci.yaml",
+    ".kokoro/.gitattributes",
+    ".kokoro/common.cfg",
+    ".kokoro/continuous/node10/common.cfg",
+    ".kokoro/continuous/node10/docs.cfg",
+    ".kokoro/continuous/node10/test.cfg",
+    ".kokoro/continuous/node12/common.cfg",
+    ".kokoro/continuous/node12/lint.cfg",
+    ".kokoro/continuous/node12/samples-test.cfg",
+    ".kokoro/continuous/node12/system-test.cfg",
+    ".kokoro/continuous/node12/test.cfg",
+    ".kokoro/docs.sh",
+    ".kokoro/lint.sh",
+    ".kokoro/populate-secrets.sh",
+    ".kokoro/presubmit/node10/common.cfg",
+    ".kokoro/presubmit/node12/common.cfg",
+    ".kokoro/presubmit/node12/samples-test.cfg",
+    ".kokoro/presubmit/node12/system-test.cfg",
+    ".kokoro/presubmit/node12/test.cfg",
+    ".kokoro/publish.sh",
+    ".kokoro/release/docs-devsite.cfg",
+    ".kokoro/release/docs-devsite.sh",
+    ".kokoro/release/docs.cfg",
+    ".kokoro/release/docs.sh",
+    ".kokoro/release/publish.cfg",
+    ".kokoro/samples-test.sh",
+    ".kokoro/system-test.sh",
+    ".kokoro/test.bat",
+    ".kokoro/test.sh",
+    ".kokoro/trampoline.sh",
+    ".mocharc.js",
+    ".nycrc",
+    ".prettierignore",
+    ".prettierrc.js",
+    "CODE_OF_CONDUCT.md",
+    "CONTRIBUTING.md",
+    "LICENSE",
+    "README.md",
+    "api-extractor.json",
+    "renovate.json",
+    "samples/README.md"
   ]
 }

